stdlib/std/math.tact

```diff
@@ -59,4 +59,18 @@ native log(num: Int, base: Int): Int;
 @name(__tact_pow)
 native pow(base: Int, exp: Int): Int;
 
-asm fun pow2(exp: Int): Int { POW2 }
\ No newline at end of file
+asm fun pow2(exp: Int): Int { POW2 }
+
+asm fun sign(x: Int): Int { SGN }
+
+// floor(x*y/2^z)
+asm fun mulrshift(x: Int, y: Int, z: Int): Int { MULRSHIFT }
```

Comment:
```suggestion
asm fun mulShiftRight(x: Int, y: Int, z: Int): Int { MULRSHIFT }
```

Not sure if that'll make it less recognizable to users that know about `MULRSHIFT` instruction, but it'll definitely make the function more [accessible to others](https://docs.tact-lang.org/book/operators/#binary-bitwise-shift-right) :)

Similar functions can be renamed in similar fashion

Reply:
let two: Int = 2;
two >> 1; // 1
4 >> 1;   // 2
5 >> 1;   // 2, due to flooring of integer values

pow(2, 254) >> 254; // 1

Reply:
done